src/domains/bizevents-app/api/transaction-service/v1/_base_policy-jwt.xml

```diff
@@ -1,92 +1,97 @@
 <policies>
     <inbound>
         <base />
-        <include-fragment fragment-id="jwt-chk-wallet-session" />
 
+            <!-- ForIO start =============== "x-continuation-token:undefined" -H "x-request-id:undefined" -->
+            <choose>
+                <when condition="@( ((string)context.Request.Headers.GetValueOrDefault("x-continuation-token","")).Equals("") || ((string)context.Request.Headers.GetValueOrDefault("x-continuation-token","")).Equals("undefined") )">
+                    <set-header name="x-continuation-token" exists-action="delete" />
+                </when>
+            </choose>
+            <choose>
+                <when condition="@( ((string)context.Request.Headers.GetValueOrDefault("x-request-id","")).Equals("") || ((string)context.Request.Headers.GetValueOrDefault("x-request-id","")).Equals("undefined"))">
+                    <set-header name="x-request-id" exists-action="delete" />
+                </when>
+            </choose>
+            <!-- ForIO stop =============== "x-continuation-token:undefined" -H "x-request-id:undefined" -->
 
-        <!-- ForIO start =============== "x-continuation-token:undefined" -H "x-request-id:undefined" -->
-        <choose>
-            <when condition="@( ((string)context.Request.Headers.GetValueOrDefault("x-continuation-token","")).Equals("") || ((string)context.Request.Headers.GetValueOrDefault("x-continuation-token","")).Equals("undefined") )">
-                <set-header name="x-continuation-token" exists-action="delete" />
-            </when>
-        </choose>
-        <choose>
-            <when condition="@( ((string)context.Request.Headers.GetValueOrDefault("x-request-id","")).Equals("") || ((string)context.Request.Headers.GetValueOrDefault("x-request-id","")).Equals("undefined"))">
-                <set-header name="x-request-id" exists-action="delete" />
-            </when>
-        </choose>
-        <!-- ForIO stop =============== "x-continuation-token:undefined" -H "x-request-id:undefined" -->
 
+            <choose>
+                <when condition="@("true".Equals("{{enable-pm-ecommerce-io}}"))"> 
```

Comment:
to be update based on the family & friends feature cc @ciuffagianluca @pietro-tota